app/templates/mandatarissen/persoon.hbs

```diff
@@ -0,0 +1,64 @@
+<AuToolbar @border="bottom" @size="large" as |Group|>
+  <Group>
+    <AuHeading @skin="2">Bewerk mandataris<br /><strong
+      >{{@model.persoon.gebruikteVoornaam}}
+        {{@model.persoon.achternaam}}</strong></AuHeading>
+  </Group>
+</AuToolbar>
+<div class="au-c-body-container au-c-body-container--scroll">
+
+  <div class="au-o-box">
+    <Person::Card @persoon={{@model.persoon}} />
+  </div>
+
+  <div class="au-o-box">
+    <AuHeading @level="3" @skin="4">
+      Huidige mandaten
+    </AuHeading>
+
+    <AuHr />
+
+    <ul class="au-c-list-vertical">
+      {{#each @model.mandatarissen as |mandataris|}}
+        {{#if mandataris.isActive}}
```

Comment:
could you reference this code in the story about adding fallbacks to these lists in the old view? LMB-189